### ISSUE TITLE
State somewhere in the README a minimal Python version needed

### ISSUE THREAD

**Issue by yarikoptic** [kaczmarj/neurodocker#139]
It seems that NeuroDocker is not compatible with python2, but also not clear which of the python3 versions should it work with? .travis.yml tests only with 3.6 -- is that the minimal supported version?  if not, would be great if it was tested on all supported versions on travis.
Cheers!

**Comment by kaczmarj** [comment]
Yes, I decided not to support Python 2. I will document the minimal 3.x version, but first I will have to find out what that is ;)

**Comment by kaczmarj** [comment]
@yarikoptic - minimum python version is 3.5.
